### PR TITLE
API Extractor support for Firestore Exp and Lite

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
This builds on top of the new script to add support for API Extractor to Firestore.

### CODE REVIEW

File: packages/firestore/exp/api-extractor.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "ae-missing-release-tag": {
+        "logLevel": "none"
+      },
+      "ae-unresolved-link": {
+        "logLevel": "none"
+      }
+    },
+    "tsdocMessageReporting": {
+      "tsdoc-undefined-tag": {
+        "logLevel": "none"
```
Comment (Feiyang1) [inline packages/firestore/exp/api-extractor.json:19]: Do you not want to get warned about APIs that don't have documentation?

**Review by Feiyang1 — APPROVED** [review]
LGTM with one question.

File: packages/firestore/exp/api-extractor.json (hunk 1 above)
Comment (schmidt-sebastian) [reply packages/firestore/exp/api-extractor.json:19]: This hides the warning for `@hideconstructor`.

**Review by schmidt-sebastian — COMMENTED** [review]
FYI I rebased this now so it has (almost) no cleanup/code changes anymore.

**Pull request merged**